### ISSUE TITLE
No ability to change the SENDER address when services send mails

### ISSUE THREAD

**Issue by lucho020** [bruceg/nullmailer#55]
@bruceg Im not sure how related these 2 issues are:
https://github.com/bruceg/nullmailer/issues/52
https://github.com/bruceg/nullmailer/issues/53

Basically, when services such as fail2ban@, root@, LinuxUserName@, send automated emails to your provider. You get bounce-backs "No Such User Here" since your provider would be using only authenticated accounts to send mail.
Instead of creating all these accounts at your provider and forwarding them..etc, id prefer it if im able to control which SENDER address i want nullmailer(sendmail binary) to use.

In other words set SENDMAIL by default to always use a sender email such as services@domain.com, therefore all services such as fail2ban, root, etc. will use that SENDER address.

This is actually a big necessity and should have already been implemented.  I believe Sendmail already has this from the start.

**Comment by jahir** [comment]
Not sure what you exactly want, but I think allmailfrom (see man nullmailer-queue) provides just that: A hard-wired envelope from address.
#52 is only about the From header, and only about the name part there, not the address itself.
#53 is about source-dependent routing, i.e. select an upstream smtp server depending on the source email address.

**Comment by lucho020** [comment]
wow.. this is exactly what i have been looking for.  THANK YOU!
Besides doing man nullmailer-queue, nowhere does it mention this allmailfrom file in any documentation of Nullmailer. And no source of information on the internet mentions that file.  I hope the documentation can be updated with this.

**Comment by jahir** [comment]
The nullmailer man page lists all control files. As usual, the HOWTO is not a comprehensive documentation.

Having said that, I also struggled to find exactly this feature and mostly found people using a wrapper script doing something like `/usr/sbin/sendmail-bin -f $(cat /etc/nullmailer/forced-from) $@ </dev/stdin` or setting _NULLMAILER_USER_/_..HOST_. It's all there in the man pages, but for some reason, we prefer a google search over the included documentation. Maybe a FAQ file could improve this...
